beam_sink/sinks/mysql.py

```diff
@@ -13,6 +13,18 @@ class MySQLConfig(BaseModel):
 
 
 class MySQLQuery(beam.PTransform):
+    """MySQLQuery.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the second line](https://app.codacy.com/manual/lislemitchell/beam-sink/pullRequest?prid=5069736)